scripts/process_biomappings.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+# import sys
+
+# biomappings_file = sys.argv[1]
+biomappings_file = "https://raw.githubusercontent.com/biopragmatics/biomappings/master/docs/_data/sssom/biomappings.sssom.tsv"
+df = pd.read_csv(biomappings_file, sep='\t')
+
+# Get only chebi to mesh rows
+df = df[(df['subject_id'].str.contains('chebi')) & (df['object_id'].str.contains('mesh'))]
+
+# Capitalize the mesh and chebi ids
+df['subject_id'] = df['subject_id'].str.upper()
```

Comment:
We should avoid string hacking like this - it can easily go wrong in the future.

https://github.com/cthoyt/curies

```
converter.standardize_curie('gomf:0032571')
```